src/utils/init.js

```diff
@@ -152,6 +158,14 @@ function validateStep(formKey, trigger = true) {
   return callback(trigger);
 }
 
+function setAuthClient(client) {
+  initState.authClient = client;
+}
+
+function getAuthClient() {
```

Comment:
out of curiosity: did you design getAuthClient to be backwards compatible with how upfront currently sets the firebase auth client? as in we aren't using a unique internal global name for the auth client, we're using the default firebase global.firebase? if so i agree with that since it means there's no immediate deprecation work necessary for upfront

Reply:
yep makes sense